Automattic/wp-calypso
# WooCommerce: Setup/NUX: Add manage egg UX

See #13612 and #13466 

To test:
- navigate to /store/:site
- ensure the dashboard appears
- ensure the 'Add a Product' button takes you /store/products/:site/add
- ensure the 'Set up shipping' button takes you to /store/settings/:site/shipping
- ensure the 'Set up payments' button takes you to /store/settings/:site/payments
- ensure the 'Set up taxes' button takes you to /store/settings/:site/tax
- ensure the 'Customize' button takes you to the customizer in wp-admin on your site

Note: The following is _NOT_ working in this PR and will be handled later/next in a separate PR:
- documentation links all point to https://support.wordpress.com/ right now since docs are not available at this time
- the footer buttons don't do anything yet, because we don't have Fledgling or Momma views of the Dashboard available yet
- the checkboxes are hard coded to all off (except customize) because we do not have those bits in state yet

cc @jameskoster for design love - I think I imported the checklist lady SVG from the sketch file improperly - the colors don't look quite right
